tests/constants/urls.js

```diff
@@ -8,6 +8,7 @@ export const APPS = {
   TASKS: 'https://my.realdevsquad.com/tasks',
   IDENTITY: 'https://my.realdevsquad.com/identity',
   MY_STATUS: 'https://my.realdevsquad.com',
+  API_BACKEND: 'https://staging-api.realdevsquad.com',
```

Comment:
shouldn't the Variable name be related to staging?

Reply:
right, i'll change it.